requirements.txt

```diff
@@ -1,2 +1,3 @@
 future
 requests
+pytz
```

Comment:
we also list requirements in the README, but I think we don't need to that when we have this file